### DISCUSSION TITLE
Step-by-step installation instructions

### DISCUSSION THREAD

**Discussion by VlaK0r** [Athou/commafeed#1077] [Q&A]
Hello!
There is step-by-step installation instructions on a  server, such as Debian?

**Comment by Athou** [comment]
There are instructions on how to start CommaFeed in the [README](https://github.com/Athou/commafeed/blob/master/README.md#deployment-on-your-own-server), do you need more information? I'd recommend using the Docker image as it's the easiest way to get started.

**Reply by VlaK0r** [reply]
I don't really understand how to work with linux. 
I have a vps server 
Debian 10 1 x 2.2 GHz, 0.5 GB RAM, 1IP	
How do I install commafeed on it?

**Reply by VlaK0r** [reply]
If installed with Docker, it will slow down a lot

**Reply by VlaK0r** [reply]
I installed docker on windows and tested commafeed. I liked it very much.

**Reply by Athou** [reply]
Docker is actually very lightweight, but if you don't want to use Docker you can follow the instructions in the README in the section "Download precompiled package" (you'll need [java](https://www.digitalocean.com/community/tutorials/how-to-install-java-with-apt-on-debian-10) too). However, I'd recommend learning a little bit of linux since you're renting a VPS.

**Reply by VlaK0r** [reply]
Okay, I'll try to figure it out

**Comment by VlaK0r** [comment]
> There are instructions on how to start CommaFeed in the [README](https://github.com/Athou/commafeed/blob/master/README.md#deployment-on-your-own-server), do you need more information? I'd recommend using the Docker image as it's the easiest way to get started.



> Docker



> There are instructions on how to start CommaFeed in the [README](https://github.com/Athou/commafeed/blob/master/README.md#deployment-on-your-own-server), do you need more information? I'd recommend using the Docker image as it's the easiest way to get started.



> Docker is actually very lightweight, but if you don't want to use Docker you can follow the instructions in the README in the section "Download precompiled package" (you'll need [java](https://www.digitalocean.com/community/tutorials/how-to-install-java-with-apt-on-debian-10) too). However, I'd recommend learning a little bit of linux since you're renting a VPS.



> Debian

Everything worked out! Thanks!
Please tell me how to update the current version?

**Comment by Athou** [comment]
It depends what route you decided to go with in the end. If you downloaded the jar file, you just need to overwrite the file with a newer version and restart the app. If you went with docker, you need to [pull the newer version, then delete and recreate the container.](https://stackoverflow.com/questions/26734402/how-to-upgrade-docker-container-after-its-image-changed)

**Reply by VlaK0r** [reply]
Thanks
